Ryujinx/Ui/RendererWidgetBase.cs

```diff
@@ -272,10 +281,56 @@ public void Initialize(Switch device)
             Renderer = Device.Gpu.Renderer;
             Renderer?.Window.SetSize(_windowWidth, _windowHeight);
 
+            if (Renderer != null)
+            {
+                Renderer.ScreenCaptured += Renderer_ScreenCaptured;
+            }
+
             NpadManager.Initialize(device, ConfigurationState.Instance.Hid.InputConfig, ConfigurationState.Instance.Hid.EnableKeyboard);
             TouchScreenManager.Initialize(device);
         }
 
+        private unsafe void Renderer_ScreenCaptured(object sender, ScreenCaptureImageInfo e)
+        {
+            if (e.Data.Length > 0)
+            {
+                Task.Run(() =>
+                {
+                    lock (this)
+                    {
+                        var    currentTime  = DateTime.Now;
```

Comment:
```suggestion
                        var    currentTime = DateTime.Now;
```
and same for other vars